Description: - Fix a typo (seperate) in manpage. [Miah]
             - Update whatis entry for manual page to something
               more useful. [Miah]
             - Remove the reference to 'info' command. [Eriberto]
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
        Miah Gregory <mace@debian.org>
Last-Update: 2014-10-05
Index: dcfldd-1.3.4.1/dcfldd.1
===================================================================
--- dcfldd-1.3.4.1.orig/dcfldd.1
+++ dcfldd-1.3.4.1/dcfldd.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.35.
 .TH DCFLDD "1" "February 2006" "dcfldd (dcfldd) 1.3.4" "User Commands"
 .SH NAME
-dcfldd \- manual page for dcfldd (dcfldd) 1.3.4
+dcfldd \- enhanced version of dd for forensics and security
 .SH SYNOPSIS
 .B dcfldd
 [\fIOPTION\fR]...
@@ -68,7 +68,7 @@ hashlog=FILE
 send MD5 hash output to FILE instead of stderr
 .IP
 if you are using multiple hash algorithms you
-can send each to a seperate file using the
+can send each to a separate file using the
 convention ALGORITHMlog=FILE, for example
 md5log=FILE1, sha1log=FILE2, etc.
 .TP
@@ -234,15 +234,3 @@ Copyright \(co 1985-2006 Free Software F
 .br
 This is free software; see the source for copying conditions.  There is NO
 warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-.SH "SEE ALSO"
-The full documentation for
-.B dcfldd
-is maintained as a Texinfo manual.  If the
-.B info
-and
-.B dcfldd
-programs are properly installed at your site, the command
-.IP
-.B info dcfldd
-.PP
-should give you access to the complete manual.
